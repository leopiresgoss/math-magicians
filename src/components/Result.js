import React from 'react';
import PropTypes from 'prop-types';

const Result = ({ result }) => (
  <div className="result">
    <h3>{result}</h3>
  </div>
);

Result.propTypes = {
  result: PropTypes.string.isRequired,
};

export default Result;
