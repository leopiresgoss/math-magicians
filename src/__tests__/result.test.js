import renderer from 'react-test-renderer';
import Result from '../components/Result';

it('renders correctly', () => {
  const screen = '';
  const tree = renderer.create(<Result result={screen} />).toJSON();
  expect(tree).toMatchSnapshot();
});
