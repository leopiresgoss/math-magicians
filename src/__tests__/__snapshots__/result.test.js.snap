// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="result"
>
  <h3>
    
  </h3>
</div>
`;
